libsolidity/formal/SolverInterface.h

```diff
@@ -132,10 +133,12 @@ class Expression
 	Expression operator()(Expression _a) const
 	{
 		solAssert(
```

Comment:
It is probably better to restructure this function to first assert that the arguments are empty, then handle the cases of int and bool function regularly and put a `solAssert(false, ...)` in the final else case.

Reply:
Yes, good idea.